tools/ak2-core.sh

```diff
@@ -483,6 +489,9 @@ patch_prop() {
   fi;
 }
 
+# grep_prop <prop name>
+grep_prop() { grep "^$1" "/system/build.prop" | cut -d= -f2; }
```

Comment:
A more accurate name might be `system_getprop` to match `file_getprop` in update-binary.

Reply:
More to the point, you can just use file_getprop.